docs/05-technical-reference/00-custom-resources/svls-01-function.md

```diff
@@ -114,6 +114,7 @@ This table lists all the possible parameters of a given resource together with t
 | **spec.buildResources.requests.cpu**          |       No       | Specifies the number of CPUs requested by the build Job's Pod to operate.       |
 | **spec.buildResources.requests.memory**       |       No       | Specifies the amount of memory requested by the build Job's Pod to operate.               |
 | **spec.runtime**                         |       No       | Specifies the runtime of the Function. The available values are `nodejs14` and `python39`. It is set to `nodejs14` unless specified otherwise.  |
+| **spec.runtimeImageOverride**                 |       No       | Specifies the runtimes image which should be used instead of default one. |
```

Comment:
```suggestion
| **spec.runtimeImageOverride**                 |       No       | Specifies the runtimes image which must be used instead of default one. |
```